contracts/native-interchain-token.clar

```diff
@@ -237,9 +250,12 @@
 (define-read-only (is-flow-limiter-raw (addr principal))
     (or
         (is-eq addr (get-its-impl))
```

Comment:
One issue with this is that the operator can not remove the ITS as a flow limiter like here:
https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/token-manager/TokenManager.sol#L113

I know that because of upgradability this is harder to do. Will check the Axelar team to see what they say.

Reply:
done